docs/tutorial/installation.rst

```diff
@@ -1,4 +1,114 @@
-Installation Guide
-========================================
+Getting Started
+===============
 
-Step by step guide on how to get started with setting up DSS.
+What you'll need
+^^^^^^^^^^^^^^^^
+
+* Ubuntu 22.04
+* Snapcraft (included in Ubuntu)
+* An internet connection
+* DSS relies on MicroK8s, that requires as little as 540MB of memory.
+  But to accommodate workloads, we recommend a system with at least 20G
+  of disk space and 4G of memory.
+
+Setting up MicroK8s
+^^^^^^^^^^^^^^^^^^^
+
+The DSS relies on a container orchestration system, that can also take
+care of exposing the host GPUs to the workloads. In this case we will use
+MicroK8s snap, which gets installed on the host machine.
+
+All the workloads and state managed by DSS will be running on top of
+MicroK8s.
+
+You can install MicroK8s with the following commands:
+
+.. code-block:: bash
+
+   sudo snap install microk8s --channel 1.28/stable --classic
+   microk8s enable storage dns rbac
+   microk8s config > ./kubeconfig
```

Comment:
Lets remove this one ... after merging @ca-scribner PR [here](https://github.com/canonical/data-science-stack/pull/113), we no longer need this instead we can just put to the step bellow: 

```
dss initialize --kubeconfig="$(microk8s kubeconfig)"
```